### PR TITLE
Cloud provider docs update

### LINKED ISSUE #2026: Documentation needs to be updated to add details on Google Cloud Storage
## Description

The documentation needs to be updated to reflect that Google Cloud Storage is now an option, in addition to S3.

[//]: # (What's it you're proposing? How should it be implemented?)

Update the relevant files

## Rationale

[//]: # (Why should this feature be implemented?)

## Use case(s) / visualization(s)

[//]: # ("Better to see something once than to hear about it a thousand times.")

### PR DESCRIPTION
**Pull request by vladdoster**
[//]: # (Thank you for helping us out: your efforts mean great deal to the project and the community as a whole!)

[//]: # (Before you proceed:)

[//]: # (1. Make sure to add yourself to `CONTRIBUTORS.rst` through this PR provided you're contributing here for the first time)
[//]: # (2. Don't forget to update the `docs/` presuming others would benefit from a concise description of whatever that you're proposing)

## Description
Updated documentation to reflect the addition of Google Cloud Storage as an option to serve static files from.

[//]: # (What's it you're proposing?)

## Rationale

[//]: # (Why does the project need that?)
So new users are aware of different development setups available from Django Cookie cutter.

## Use case(s) / visualization(s)
A user wants to create a new project from Django Cookiecutter. They will now see that they can choose between AWS and GCS as a provider for serving static files.

[//]: # ("Better to see something once than to hear about it a thousand times.")

Fixes #2026

### CODE REVIEW

**Comment by vladdoster** [comment]
See #2026 for issue

File: docs/settings.rst [hunk 1]
```diff
@@ -45,6 +45,7 @@ DJANGO_AWS_ACCESS_KEY_ID                AWS_ACCESS_KEY_ID           n/a
 DJANGO_AWS_SECRET_ACCESS_KEY            AWS_SECRET_ACCESS_KEY       n/a                                            raises error
 DJANGO_AWS_STORAGE_BUCKET_NAME          AWS_STORAGE_BUCKET_NAME     n/a                                            raises error
 DJANGO_AWS_S3_REGION_NAME               AWS_S3_REGION_NAME          n/a                                            None
+DJANGO_GCE_STORAGE_BUCKET_NAME          GS_BUCKET_NAME              n/a                                            raises error
```
Comment (browniebroke) [inline docs/settings.rst]: I think we should also document the environment variable `GOOGLE_APPLICATION_CREDENTIALS` with Django setting n/a.

https://github.com/pydanny/cookiecutter-django/pull/2012/files#diff-78868204609793ad60885e85e4a879e6

```suggestion
DJANGO_GCE_STORAGE_BUCKET_NAME          GS_BUCKET_NAME              n/a                                            raises error
GOOGLE_APPLICATION_CREDENTIALS          n/a                         n/a                                            raises error
```

File: docs/project-generation-options.rst [hunk 2]
```diff
@@ -63,6 +63,12 @@ js_task_runner:
     1. None
     2. Gulp_
 
+cloud_provider:
+    Select a cloud provider for static files. The choices are:
```
Comment (browniebroke) [inline docs/project-generation-options.rst]: I suggest extending this to media as well. They mean 2 different things in the Django space:

```suggestion
    Select a cloud provider for static & media files. The choices are:
```

File: docs/settings.rst (hunk 1 above)
Comment (vladdoster) [reply docs/settings.rst]: Good call

**Comment by vladdoster** [comment]
Thank you for the feedback! I committed both of your recommendations. :+1:

**Comment by browniebroke** [comment]
Thanks again 🎉

**Pull request merged**